test/rust_src/src/keymap-tests.el

```diff
@@ -253,6 +253,31 @@
     (should (equal (current-global-map) '(keymap (3 keymap (26 . emacs-version)))))
     (use-global-map backup-keymap)))
 
+(ert-deftest test-command-remapping ()
+
+
+  )
+
+
+(defun my-kill-line ())
+
+(make-keymap 'my-mode-map)
+
+
+(defvar my-mode-map
+  (let ((map (make-keymap)))
+    (set-keymap-parent map text-mode-map)
+    map))
+
+
+(define-key my-mode-map [remap kill-line] 'my-kill-line)
+
+
+(command-remapping 'kill-line)
+
+(define-key my-mode-map [remap kill-line] 'my-kill-line)
+
+
```

Comment:
The whitespace looks very strange in the changes in this file, and the body of test-command-remapping is empty. Is this part unfinished?

Reply:
Oops, I pushed the wrong branch.